tests/components/homeassistant/test_init.py

```diff
@@ -314,6 +314,14 @@ async def test_setting_location(hass: HomeAssistant) -> None:
     assert len(events) == 1
     assert hass.config.latitude == 30
     assert hass.config.longitude == 40
+    assert hass.config.elevation != 50
```

Comment:
I would move this assertion before line 308 and modify it slightly:
```python
elevation = hass.config.elevation
assert elevation != 50
```

Now, we can assert here that the elevation was not changed if we don't set it.
```suggestion
    assert hass.config.elevation == elevation
```

Reply:
Done, thanks.